paint/PAINT_Ellipse.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include "PAINT_Shape.h"
+
+namespace paint
+{
+	class Ellipse final :
+		public Shape
+	{
+	public:
+		Ellipse() = default;
+		virtual ~Ellipse() = default;
+		Ellipse(const Ellipse& that) = default;
+		Ellipse(Ellipse&& that) = default;
+		Ellipse& operator=(const Ellipse& that) = default;
+		Ellipse& operator=(Ellipse&& that) = default;
+
+		std::vector<gfx::Line> shapeGeometry(win::Coords startCoords, win::Coords currentCoords) override;
+		std::vector<int> ellipse(int x, int a, int b, int h, int k) const;
```

Comment:
This doesn't need to be a member function. Make it static in the cpp file.
**SHOULD**

Reply:
done